tsconfig.json

```diff
@@ -27,11 +27,11 @@
       ]
     },
     "types": [
+      "@types/node",
       "@nuxt/types",
+      "@nuxtjs/sentry",
```

Comment:
Why changing the order?

Reply:
it seems that the type below (axios, i18n) extends the type above (node => nuxt) so the order some how matters